src/frontend/js/modules/application-menu.js

```diff
@@ -11,7 +11,7 @@ angular.module('esn.application-menu', [])
       replace: true,
       template: require('../../views/modules/application-menu/application-menu-toggler.pug'),
       link: function($scope) {
-        $scope.appGridItems = process.env.APP_GRID_ITEMS;
+        $scope.appGridItems = window.openpaas.APP_GRID_ITEMS || process.env.APP_GRID_ITEMS;
```

Comment:
Are you sure the order here ?

Reply:
yes, the dynamic `openpaas.js` file should have priority ( the value can be changed on the fly ) whereas the process.env variable is kinda static fallback value.